### PR TITLE
Remove schema definition from 'bde_TablesAffected' function (closes #130)

### LINKED ISSUE #130: bde_control.bde_TablesAffected function is incorrectly populated with <schema>.<table_name>
bde_control.bde_TablesAffected function is incorrectly populated with <schema>.<table_name>

for example in

https://github.com/linz/linz-lds-bde-schema/blob/00d4959ea15196f9c957f0bb466022b0dd97b0d0/sql/04-lds_layer_functions.sql.in#L1555

but this function doesn't expect schema being provided. This result that function always returns False

/cc @strk

### PR DESCRIPTION
**Pull request by imincik**
`bde_control.bde_TablesAffected` function doesn't expect schema definition. If schema is provided, function returns always False.

Closes #130

### CODE REVIEW

**Comment by strk** [comment]
Good catch!
I think we should patch the documentation for the `bde_TablesAffected` to specify
that the `bde` schema is assumed, which is otherwise unspecified:

from `perldoc LINZ::BdeUpload`:
```
   INT bde_control.bde_TablesAffected( upload_id INT, tables name[], test TEXT )
        Tests tables that are affected by an upload. tables is a list of tables to check.
```

**Comment by strk** [comment]
For the record, the inclusion of schema was added with 3fcfbee640d9c5a1c1422fd27a34b42bc8cbef12 as part of a change aimed at removing enforcement of a specific `search_path` variable in the session loading the scripts. The change of those `name` typed function arguments was probably an overzealous global search & replace

**Comment by strk** [comment]
I've ticketed documentation issue in https://github.com/linz/linz-bde-uploader/issues/173

**Comment by strk** [comment]
Ideally we'd have a testcase for this but currently we do NOT test the simplified layers creation functions AT ALL, I'm afraid (see https://github.com/linz/linz-lds-bde-schema/issues/18)

**Comment by imincik** [comment]
@strk , changelog updated.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ this file.
   from `bde`.`crs_sys_code`. Affects all "lds.title*" tables. (#125)
 ### Enhanced
 - Create no revision when versioning empty tables (#117)
+### Fixed
+- Fixed `bde_TablesAffected` function (remove schema deffinition from table
```
Comment (strk) [inline CHANGELOG.md]: typo (deffinition) -- I still think we want bugfixes to go to stable branches, not feature branches (ie: 1.3 for 1.3.1)

**Comment by imincik** [comment]
Typo fixed

**Comment by imincik** [comment]
> I still think we want bugfixes to go to stable branches, not feature branches (ie: 1.3 for 1.3.1)

I am not sure what you mean here. I am merging to `master` branch. I am not against merging this patch to stable branch (BTW, that's why I am not updating CHANGELOG to make this backport easier)

**Comment by strk** [comment]
Yes, now we can merge, although at this point the changelog entry for the `bde.` prefix removal would not belong here anymore (not a big deal to keep it)

**Comment by strk** [comment]
Travis confirmed the fix was good in 1.3 branch: https://travis-ci.com/linz/bde-processor-deployment/builds/90211470

**Pull request merged**